src/analyses/base.ml

```diff
@@ -100,6 +100,12 @@ struct
     | 0 -> Flag.compare x2 y2
     | x -> x
 
+  let get_ikind t = match Cil.unrollType t with TInt (ik,_) | TEnum ({ekind = ik; _},_) -> ik | _ ->
```

Comment:
This is duplicated in base, arrayDomain and valueDomain.

Reply:
Moved it into `Cilfacade`